src/scroll-timeline-css.js

```diff
@@ -41,8 +41,26 @@ function initMutationObserver() {
     el.innerHTML = newSrc;
   }
 
-  function handleLinkedStylesheet(el) {
-    // TODO
+  function handleLinkedStylesheet(linkElement) {
+    // Filter only css links to external stylesheets.
+    if (linkElement.type != 'text/css' && linkElement.rel != 'stylesheet' || !linkElement.href) {
+      return;
+    }
+    const url = new URL(linkElement.href, document.baseURI);
+    if (url.origin != location.origin) {
+      // Most likely we won't be able to fetch resources from other origins.
+      return;
+    }
+    fetch(linkElement.getAttribute('href')).then(async (response) => {
+      const result = await response.text();
```

Comment:
Is it worthwhile to defer transpilation to a new task with a setTimeout to avoid blocking if input or other operations are in progress when the fetch completes?

Reply:
I'm happy to put a `setTimeout(transpileFunction, 0)` in until we have a reason that it is necessary to transpile as soon as the request is done if you think this will be important.

Reply:
Yeah not critical here, I think I always try to have a performance outlook and know these requests happen pretty early in the user experience. It might definitely be a premature optimization though.